### PR TITLE
Example/openpose

### PR DESCRIPTION
**Pull request by fx-lange**
hi @cvalenzuela,

just finished a proof of concept example connecting RunwayML with Unity running the OpenPose model. I'm using OSC for the connection. It is basically ported from the openFrameworks example and only drawing parts and connections as gizmos.

Please have a look and let me know if this works for you. My mac doesn't perform to well so I could only test it with a data rate of 2 samples per second. Couldn't test it on my PC yet.

I'm working on the Unity 2018.1 Beta. I could downgrade to 2017 if you like but 2018 should be out of beta in the next couple of days. 

I'm planning to pretty up the drawing and include a camera feed to make the example more meaningful in the next days. 

Let me know if you have any questions

Best,
Felix

### CODE REVIEW

**Comment by fx-lange** [comment]
I should probably mention that I'm using an [UnityOSC fork](https://github.com/fx-lange/UnityOSC) and [Json.Net.Unity3D](https://github.com/SaladLab/Json.Net.Unity3D) for OSC and JSON support. Both are already integrated in the Asset folder.

File: openpose/Assets/Scripts/OSCRunwayBridge.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections;
```
Comment (cvalenzuela) [inline openpose/Assets/Scripts/OSCRunwayBridge.cs]: Can you add a license? and you as a contributor? See: https://github.com/runwayml/examples_openFrameworks/blob/master/openpose/src/ofApp.h

File: openpose/Assets/Scripts/OpenPoseDebugDraw.cs [hunk 2]
```diff
@@ -0,0 +1,134 @@
+﻿using System.Collections;
```
Comment (cvalenzuela) [inline openpose/Assets/Scripts/OpenPoseDebugDraw.cs]: Same as OSCRunwayBridge.cs!

**Review by cvalenzuela — COMMENTED** [review]
wow, this is great! thanks for putting this up @fx-lange!
I tried and it works like a charm!

screen shot 2018-04-28 at 15 52 20

I just left two comments related to the license and attributions.

Can you also move this examples inside its own folder. Something like this:
+ OpenPose
  - receiveOnly

Like the JS examples here: https://github.com/runwayml/examples_javascript/tree/master/openpose

So then the camera feed example can go into it's own folder too, and people can choose which one to use.

thanks a lot!

**Comment by fx-lange** [comment]
Sure, will do.

For Unity it is probably also fine to have similar examples in the same project as different Scenes. At least thats how I would handle the version with WebCam feed. Its a different Scene setup but the same code base.

Btw. do you want to wait for the WebCam version or have it as another pull request?

**Comment by cvalenzuela** [comment]
Yes, that sounds good! same project, different Scenes.

I guess we could move on and merge this and have the webcam as another PR if that's ok with you!

**Comment by fx-lange** [comment]
Yes, awesome - go ahead.

**Comment by fx-lange** [comment]
@cvalenzuela the webcam integration is now also ready, should I add it here or as an additional pull request?

**Comment by cvalenzuela** [comment]
sure! let's added it here

**Comment by fx-lange** [comment]
@cvalenzuela just merged into this. happy testing and merging (hopefully)

**Comment by cvalenzuela** [comment]
looks perfect! this is great!

thanks, @fx-lange, merging now!

**Pull request merged**